scripts/linux/electron-installer-appimage.sh

```diff
@@ -105,7 +105,6 @@ fi
 if [ -z "$TMPDIR" ]; then
   TMPDIR=$(mktemp -d)
```

Comment:
Sounds good to me. Do you think we should still rename this in the Makefile, though? Using a custom environment variable means that we have to document it, and we still have to include an `if` conditional to ensure it exists.

If we only rename on the script, then it means we can simply do:

```sh
TEMPORARY_DIR=$(mktemp -d)
```

and get rid of the conditional. Since we still pass `TMPDIR` in the Makefile, `mktemp` will create a directory inside it, and get still get exactly what we wanted.